processing/src/main/java/io/druid/query/filter/DimFilterCacheHelper.java

```diff
@@ -37,7 +37,7 @@
   static final byte JAVASCRIPT_CACHE_ID = 0x7;
   static final byte SPATIAL_CACHE_ID = 0x8;
   static final byte IN_CACHE_ID = 0x9;
-  static final byte STRING_SEPARATOR = (byte) 0xFF;
+  public static final byte STRING_SEPARATOR = (byte) 0xFF;
```

Comment:
I needed to use STRING_SEPARATOR and dint want to create another one. should I move it to some other helper class?